lib/event_class.rb

```diff
@@ -0,0 +1,30 @@
+require 'pry'
+
+class Event
+attr_reader   :name
+              :ages
+
+  def initialize(curling, ages)
```

Comment:
While this will _work_ the event will not always be curling, so try to use more descriptive variable names like this:

```ruby
def initialize(event_name, ages)
  @name = event_name
  @ages = ages
end
```

This will also allow you to use the `:name` attr_reader you have defined above.